tests/test_data/output_VerticalHangingFormatter.txt

```diff
@@ -0,0 +1,31 @@
+"""Docstring here"""
+
+from __future__ import print_function, unicode_literals
+import datetime
+from os import path as ospath
+
+import something  # with comment
+from a import b
+from a.b import c, d
+from other import (  # noqa
+                   # lots of happy things below
+                   and_rainbows,
+                   something,  # something comment
+                   something_else)
+from other.package.subpackage.module.submodule import (  # noqa
```

Comment:
is this desired? maybe:

```python
from other.package.subpackage.module.submodule import (CONSTANT,  # noqa
                                                       Klass,
                                                       bar,
                                                       foo,
                                                       rainbows)
```

 assuming if `CONSTANT` does not have a comment of its own. If yes then current implementation is indeed better.

I dont really care since Im not a fan of this style so will leave up to you